src/common/utils/validator/displayNameValidator.ts

```diff
@@ -1,9 +1,10 @@
 import * as yup from 'yup';
 import { MessageWithPayload } from '../../../domain/shared/i18n/ValidationMessageTranslation';
+import { SMALL_TEXT_LENGTH } from '../../../models/constants/field-length.constants';
 
 export const displayNameValidator = yup
   .string()
   .test('is-not-spaces', 'forms.validations.nonBlank', value => !value || !/^[\s]*$/.test(value))
   .required('forms.validations.required')
```

Comment:
I am pretty sure the the above two lines use translation strings and the hook needs to be used.